src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -268,7 +269,9 @@ const MetaTraderUI = (() => {
         if (action === 'cashier') { // Manage Fund
             const client_currency = Client.get('currency');
             const mt_currency     = MetaTraderConfig.getCurrency(acc_type);
+            const transfer_fee = isStableCoin(client_currency) ? '0.5%' : '1%';
```

Comment:
could you align by `=` similar to the above lines?

Reply:
sure